prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -46,6 +46,44 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: Role
+    metadata:
+      name: weave-net2
```

Comment:
nit: Is there a reason the permissions aren't just added to the existing role?
If they should be separate, a more descriptive name would be nice (`weave-net-peers`?)